src/pudl/metadata/codes.py

```diff
@@ -2391,3 +2296,103 @@
         "ignored_codes": [],
     },
 }
+
+# The entity type codes were never fully reconciled. Preserving this work for reference.
+# See https://github.com/catalyst-cooperative/pudl/issues/1392
+DISABLED_CODE_METADATA = {
```

Comment:
I didn't want to just delete this because it has useful information, but it's not finished.

Reply:
Oh yay thank you for dealing with this